docs: docs for remote joins (close #4911) (#5132)
hasura/graphql-engine: docs/graphql/manual/schema/relationships/action-relationships.rst

.. meta::
   :description: Adding an action relationship with Hasura
   :keywords: hasura, docs, action relationship, remote join

.. _add_action_relationship:

Creating action relationships
=============================

.. contents:: Table of contents
  :backlinks: none
  :depth: 1
  :local:

Introduction
------------

> :ref:`Actions <actions>` are a way to extend Hasura’s schema with custom business logic using custom queries and mutations. They can be used to call a REST endpoint from Hasura as a resolver for custom types. They are especially useful for setting up serverless functions as resolvers.

Reviewer: ```suggestion
:ref:`Actions <actions>` are a way to extend Hasura’s schema with custom business logic using custom queries and mutations. The resolvers for these custom fields are written in REST endpoints. They are especially useful for setting up serverless functions as resolvers.
```
